src/main/resources/assets/tconstruct/models/block/table/chest/chest.json

```diff
@@ -1,258 +1,123 @@
-        {
-            "name": "Drawer - Base - Bottom",
-            "from": [4, 5, 1],
-            "to": [12, 6, 11],
-            "shade": true,
-            "faces": {
-                "east": {"uv": [5, 0, 15, 1], "texture": "#drawer_side"},
-                "west": {"uv": [1, 0, 11, 1], "texture": "#drawer_side"},
-                "up": {
-                    "uv": [0, 5, 8, 15],
-                    "texture": "#drawer_side",
-                    "rotation": 180
-                },
-                "down": {"uv": [0, 5, 8, 15], "texture": "#drawer_side"}
-            }
-        },
-        {
-            "name": "Drawer - Side - Bottom Right",
-            "from": [4, 6, 1],
-            "to": [5, 8, 11],
-            "shade": true,
-            "faces": {
-                "east": {"uv": [5, 0, 15, 2], "texture": "#drawer_side"},
-                "west": {"uv": [1, 0, 11, 2], "texture": "#drawer_side"},
-                "up": {"uv": [0, 1, 1, 11], "texture": "#drawer_side"}
-            }
-        },
-        {
-            "name": "Drawer - Side - Bottom Left",
-            "from": [11, 6, 1],
-            "to": [12, 8, 11],
-            "shade": true,
-            "faces": {
-                "east": {"uv": [5, 0, 15, 2], "texture": "#drawer_side"},
-                "west": {"uv": [1, 0, 11, 2], "texture": "#drawer_side"},
-                "up": {"uv": [0, 1, 1, 11], "texture": "#drawer_side"}
-            }
-        },
-        {
-            "name": "Drawer - Front - Bottom",
-            "from": [3, 5, 0],
-            "to": [13, 9, 1],
-            "shade": true,
-            "faces": {
-                "north": {"uv": [3, 2, 13, 6], "texture": "#drawer_front"},
-                "east": {"uv": [3, 2, 4, 6], "texture": "#drawer_front"},
-                "west": {"uv": [12, 2, 13, 6], "texture": "#drawer_front"},
-                "up": {"uv": [3, 2, 13, 3], "texture": "#drawer_front"},
-                "down": {
-                    "uv": [3, 5, 13, 6],
-                    "texture": "#drawer_front",
-                    "rotation": 180
-                }
-            }
-        },
-        {
-            "name": "Drawer - Handle - Bottom",
-            "from": [7.5, 6.5, -0.5],
-            "to": [8.5, 7.5, 0],
-            "shade": true,
-            "faces": {
-                "north": {"uv": [8, 12, 9, 13], "texture": "#drawer_front"},
-                "east": {"uv": [7, 12, 8, 13], "texture": "#drawer_front"},
-                "west": {"uv": [9, 12, 10, 13], "texture": "#drawer_front"},
-                "up": {"uv": [7, 11, 8, 12], "texture": "#drawer_front"},
-                "down": {"uv": [6, 12, 7, 13], "texture": "#drawer_front"}
-            }
-        }
-    ]
-}
+	"parent": "block/block",
+	"textures": {
+		"0": "tconstruct:block/table/chest/drawer_front",
+		"1": "tconstruct:block/table/chest/drawer_side",
+		"2": "tconstruct:block/table/chest/drawer_top",
+		"3": "tconstruct:block/table/chest/drawer_bottom",
```

Comment:
None of these textures need to be in the base block, you just override them in a child. Also, bring back the particle alias, there is no reason we need to define that twice.